src/Microsoft.ML.Transforms/Text/TextTransform.cs

```diff
@@ -20,11 +20,11 @@
 using System.Linq;
 using System.Text;
 
-[assembly: LoadableClass(TextTransform.Summary, typeof(IDataTransform), typeof(TextTransform), typeof(TextTransform.Arguments), typeof(SignatureDataTransform),
-    TextTransform.UserName, "TextTransform", TextTransform.LoaderSignature)]
+[assembly: LoadableClass(FeaturizeTextEstimator.Summary, typeof(IDataTransform), typeof(FeaturizeTextEstimator), typeof(FeaturizeTextEstimator.Arguments), typeof(SignatureDataTransform),
+    FeaturizeTextEstimator.UserName, "FeaturizeTextEstimator", FeaturizeTextEstimator.LoaderSignature)]
 
-[assembly: LoadableClass(TextTransform.Summary, typeof(ITransformer), typeof(TextTransform), null, typeof(SignatureLoadModel),
-    TextTransform.UserName, "TextTransform", TextTransform.LoaderSignature)]
+[assembly: LoadableClass(FeaturizeTextEstimator.Summary, typeof(ITransformer), typeof(FeaturizeTextEstimator), null, typeof(SignatureLoadModel),
+    FeaturizeTextEstimator.UserName, "FeaturizeTextEstimator", FeaturizeTextEstimator.LoaderSignature)]
 
 namespace Microsoft.ML.Runtime.Data
```

Comment:
>Microsoft.ML.Runtime.Data [](start = 10, length = 25)

@Zruty0 i asked on the bug, but do we want to move those from this namespace to Microsoft.ML.Transforms ?

Reply:
And I answered on the bug. Yes, we don't want `Runtime` in the namespaces. This particular one should probably go to `Microsoft.ML.Transforms.Text`.

Ultimately, we want the directory structure to reflect the namespace structure, where possible

---
In reply to: [226817844](https://github.com/dotnet/machinelearning/pull/1328#discussion_r226817844) [](ancestors = 226817844)